stout/flags.cc

```diff
@@ -101,6 +188,34 @@ void Parser::Parse(int* argc, const char*** argv) {
 
     // Skip anything that doesn't look like a flag.
     if (arg.find("--") != 0) {
+      if (subcommands_.count(arg) > 0) {
```

Comment:
But duplicate subcommands are ok as long as they are nested! Not sure I'm understanding what this is catching?